tags.go

```diff
@@ -26,10 +26,16 @@ func DefaultTagSuffix(ref, suffix string) []string {
 
 // DefaultTags returns a set of default suggested tags based on
 // the commit ref.
-func DefaultTags(ref string) []string {
+func DefaultTags(ref, commitBranch, defaultBranch string) []string {
```

Comment:
I think we can remove `commitBranch` since this information is already passed in the `ref` field.

Reply:
Done. https://github.com/drone-plugins/drone-docker/pull/160/commits/0927e34a03293409b1de12926974fd125083e7ce